CONTRIBUTING.md

```diff
@@ -392,7 +392,7 @@ Before a PR can be merged, it should satisfy the following:
     - [ ] Check if `make distclean; git status` shows any files that were generated from the source by the project (not an IDE) but we don't want to track in the repository.
     - [ ] Add new patterns (just for the new files above) and re-run the above test.
 - [ ] New examples:
-    - [ ] Add sample runs at the top of the example and make sure they work.
```

Comment:
@tzanio Since this is for new examples, I wanted to emphasize that new examples should have sample runs. Just so you know.
No big deal.

Reply:
Understood. I thought that it is more important to emphasize that *all* sample runs should be tested then that there should be sample runs in a first places... that goes without saying :)